src/main/java/uk/gov/digital/ho/pttg/application/namematching/NameMatchingFunctions.java

```diff
@@ -3,13 +3,11 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-import static com.google.common.collect.Iterables.concat;
-import static com.google.common.collect.Lists.newArrayList;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 public class NameMatchingFunctions {
 
-    public static List<String> splitIntoDistinctNames(String name) {
+    static List<String> splitIntoDistinctNames(String name) {
```

Comment:
As this is now only used by InputNames - and I think that's a good thing - perhaps this method should become a private method of InputNames?

Reply:
I considered that, same for some other functions.  Let's discuss.